python/pyspark/pandas/tests/indexes/test_datetime.py

```diff
@@ -269,6 +256,10 @@ def test_map(self):
         mapper_pser = pd.Series([1, 2, 3], index=pidx)
         self.assert_eq(psidx.map(mapper_pser), pidx.map(mapper_pser))
 
+    def test_isocalendar(self):
```

Comment:
If we want to add a new test, then I think it's better to also move the related tests from "test_properties" to here.

Reply:
Moved to the appropriate test